migration/1714018700116-add_archived_QFRound_fields.ts

```diff
@@ -0,0 +1,29 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddArchivedQFRoundFields1714018700116
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+          ALTER TABLE "qf_round" 
+          ADD COLUMN IF NOT EXISTS "bannerBgImage" character varying
+        `);
+
+    await queryRunner.query(`
+          ALTER TABLE "qf_round" 
+          ADD COLUMN IF NOT EXISTS "sponsorsImgs" character varying[] DEFAULT '{}' NOT NULL
+        `);
+  }
```

Comment:
You can do the queries in 1 transaction. By adding the next command in the next line